### PR TITLE
Fix ann-bench Documentation

### PR DESCRIPTION
**Pull request by divyegala**
- In `run.py` instead of hard-coding `base.fbin` and `query.fbin`, let conf files specify the filenames
- In `data_export.py` instead of requiring `<result_filepaths>`, the script checks `<dataset_path/result` automatically

### CODE REVIEW

File: docs/source/raft_ann_benchmarks.md
Comment (cjnolet) [inline docs/source/raft_ann_benchmarks.md:47]: Is this `result/<dataset>`? I thought we changes this to `bench/ann/data/<dataset>/result` but we talked about computing this instead of having the user pass it in.

File: docs/source/raft_ann_benchmarks.md
Comment (cjnolet) [inline docs/source/raft_ann_benchmarks.md:185]: I think we need to add `bench/ann` to the README.md as well.

Comment (divyegala) [reply docs/source/raft_ann_benchmarks.md:185]: Where to add it and in what capacity?

Comment (cjnolet) [reply docs/source/raft_ann_benchmarks.md:183]: Bottom of readme.md where we list all the paths in the codebase and what they are for.

**Review by cjnolet — APPROVED** [review]

**Comment by cjnolet** [comment]
Waiting on readme updates before merge (we probably should have made those updates when we moved everything into bench/ann)

File: docs/source/raft_ann_benchmarks.md
Comment (divyegala) [reply docs/source/raft_ann_benchmarks.md:183]: Done

**Pull request merged**
